devtools/create_cryo/eden_static.h

```diff
@@ -20,12 +20,10 @@
  *
  */
 
-#include "cryo/defs.h"
-#include "cryo/cryolib.h"
+#pragma once
+#include "eden.h"
 
-namespace Cryo {
-
-Follower followerList[] = {
+Follower followerList[15] = {
 //            char,                 X,  sx, sy,  ex,  ey,bank,
 	{ PersonId::pidGregor,          5, 211,  9, 320, 176, 228,   0,  0 },
```

Comment:
Thanks for that, the namespaces around the enums looked strange, but now I understood why they are there. Strangely, this syntax for enums works fine for mingw64, so I assume that newer versions of GCC accept it. Anyway, I've wrapped all enums around namespaces, like the rest of the code